### PR TITLE
Address modal ui2 design.

### PR DESCRIPTION
**Pull request by andreyvEze**
iPhone 12 address modal
touch address modal
iPad address modal_

### CODE REVIEW

File: src/components/themed/ThemedButtons.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ function spacingStyles(props: Props, theme: Theme) {
 const getStyles = cacheStyles((theme: Theme) => {
   const commonButton = {
     alignItems: 'center',
-    borderRadius: theme.rem(1.5),
+    borderRadius: theme.rem(0.25),
```
Comment (andreyvEze) [inline src/components/themed/ThemedButtons.js:172]: create new themed button for it, just for this PR change. then create second PR that removes this and changes borderRadius

File: src/components/modals/AddressModal.js [hunk 2]
```diff
@@ -323,80 +347,83 @@ class AddressModalConnected extends React.Component<Props, State> {
 
   handleClose = () => this.props.bridge.resolve(null)
   keyExtractor = (item: string, index: number) => index.toString()
-  render() {
+
+  renderTracker = (keyboardAnimation, keyboardLayout) => {
```
Comment (andreyvEze) [inline src/components/modals/AddressModal.js:351]: remove paste button

File: src/components/modals/AddressModal.js [hunk 3]
```diff
@@ -323,80 +347,83 @@ class AddressModalConnected extends React.Component<Props, State> {
...
-                  label={statusLabel}
-                  onSubmitEditing={this.handleSubmit}
-                  inputAccessoryViewID={inputAccessoryViewID}
-                />
+      <ThemedModal bridge={this.props.bridge} onCancel={this.handleClose} paddingRem={1}>
+        <ModalTitle center paddingRem={[0, 2, 1]}>
+          {title || s.strings.address_modal_default_header}
+        </ModalTitle>
+        <View style={styles.container}>
+          <KeyboardTracker>{this.renderTracker}</KeyboardTracker>
```
Comment (andreyvEze) [inline src/components/modals/AddressModal.js:378]: remove

File: src/components/modals/AddressModal.js [hunk 4]
```diff
@@ -323,80 +347,83 @@ class AddressModalConnected extends React.Component<Props, State> {
...
+          {Platform.OS === 'ios' ? (
```
Comment (andreyvEze) [inline src/components/modals/AddressModal.js:379]: remove this

File: src/components/modals/AddressModal.js [hunk 5]
```diff
@@ -323,80 +347,83 @@ class AddressModalConnected extends React.Component<Props, State> {
...
+            onBlur={this.fieldOnBlur}
+            value={uri}
+            onClear={this.clearText}
+            isClearable={isFocused}
+            marginRem={[0, 1]}
+            ref={this.textInput}
+            error={fieldError}
+            inputAccessoryViewID={inputAccessoryViewID}
+            blurOnSubmit
+            // style={addressInputStyles}
```
Comment (andreyvEze) [inline src/components/modals/AddressModal.js:407]: remove

File: src/components/themed/ModalParts.js [hunk 6]
```diff
@@ -27,11 +28,22 @@ export function ModalCloseArrow(props: { onPress: () => void }) {
 
   return (
     <TouchableOpacity onPress={props.onPress} style={styles.closeArrow}>
-      <AntDesignIcon name="down" size={theme.rem(1.25)} color={theme.secondaryText} />
+      <FontAwesome5 name="chevron-down" size={theme.rem(1.25)} color={theme.iconTappable} />
```
Comment (andreyvEze) [inline src/components/themed/ModalParts.js:31]: change it back

File: src/components/themed/EdgeTextField.js [hunk 7]
```diff
@@ -67,7 +67,7 @@ class EdgeTextFieldOutlinedComponent extends React.PureComponent<EdgeOutlinedTex
           {...rest}
         />
         {isClearable && (
-          <TouchableOpacity onPress={this.clearText} style={styles.outlinedTextFieldClearContainer}>
+          <TouchableOpacity onPress={this.clearText} style={[styles.outlinedTextFieldClearContainer, { marginRight: spacings.marginRight }]}>
```
Comment (andreyvEze) [inline src/components/themed/EdgeTextField.js:70]: add search icon

File: src/components/themed/ModalParts.js (hunk 6 above)
Comment (thehobbit85) [reply src/components/themed/ModalParts.js:31]: Ignore this comment.
Julia approved this change.

File: src/components/themed/ThemedButtons.js (hunk 1 above)
Comment (thehobbit85) [reply src/components/themed/ThemedButtons.js:172]: Ignore this comment.
Julia approved this change.

File: src/components/themed/EdgeTextField.js [hunk 8]
```diff
@@ -51,14 +52,31 @@ class EdgeTextFieldOutlinedComponent extends React.PureComponent<EdgeOutlinedTex
     }
   }
 
+  showSearchIcon = () => {
+    const { fieldRef, value } = this.props
+    if (fieldRef && fieldRef.current) {
+      if (fieldRef.current.focused) return false
+    }
+    return !value
```
Comment (andreyvEze) [inline src/components/themed/EdgeTextField.js:60]: change to `value != null`

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**